test/common/testlib.py

```diff
@@ -672,14 +683,14 @@ def open_session_menu(self):
             self.click("#toggle-menu")
 
     def open_superuser_dialog(self):
-        if self.cdp.mobile:
+        if self.current_layout["is_mobile"]:
```

Comment:
There is only three places which neeed `is_mobile` now. Could we replace this with a `self.is_mobile()` function which checks the DOM and returns the current mode?

Reply:
I think we can remove `is_mobile` completely.  If necessary, it can be a optional argument to `open_superuser_dialog`.  Tests do know exactly in which layout they are.